src/app/shared/components/upcoming-appointments/upcoming-appointments.component.ts

```diff
@@ -29,14 +28,47 @@ export class UpcomingAppointmentsComponent implements OnInit {
 
   getSchedules(e) {
     const convertedDate = convertToJavaLocalDateTime(e);
```

Comment:
```suggestion
    const convertedDate = convertToJavaLocalDateTime(e);
```

que nome de função é essa? kkkkk